### ISSUE TITLE
getting negative loss

### ISSUE THREAD

**Issue by ni1lloc** [HawkAaron/warp-transducer#34]
Hello, 
I'm trying to feed some random matrix into the loss function, and then negative loss returned.
![image](https://user-images.githubusercontent.com/17943172/62607783-35e13b00-b931-11e9-9e65-60891a5afe52.png)
tf1.9/cuda9.0/cudnn7.1
What's the normal range of the loss value? Or, any suggestions in debugging? Thanks a lot!

**Comment by HawkAaron** [comment]
The input activation tensor shaped [B, T, U+1, V], and U is label length; 

Please make sure using the GPU version.

Please check the [tf python doc](https://github.com/HawkAaron/warp-transducer/blob/master/tensorflow_binding/README.md#python-interface) for more details about tensor shapes.

**Comment by ni1lloc** [comment]
Thanks!
